galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -396,3 +396,12 @@ messages:
     args:
       - name: gateways
         type: "[]string"
+
+  - name: "ServiceEntryHTTPSTrafficOnHTTPPort"
+    code: IST0139
+    level: Error
```

Comment:
I think at most this should be a warning.

Reply:
will change to Warning, I think this makes sense. Thanks

Reply:
change to warning.